.ci/scripts/install-msitools.sh

```diff
@@ -0,0 +1,4 @@
+set -euo pipefail
```

Comment:
IIUC, we said `.ci/scripts/install-msitools.sh` will be renamed to `.buildkite/scripts/install-msitools.sh` and then all the references for `.ci/scripts/install-msitools.sh` will be changed for `.buildkite/scripts/install-msitools.sh`, similarly done for `.buildkite/x-pack/pipeline.xpack.osquerybeat.yml` that will use `.buildkite/scripts/install-msitools.sh`